dluhc-component-library/src/components/timetableForm/components/DateInput.tsx

```diff
@@ -0,0 +1,22 @@
+const DateInput = () => {
```

Comment:
`Non-blocker` Now that we have more than one form, we should probably pull the none specific sub components to the normal component level. Eg `Textarea` and `DateInput` will probably be useful in both.

Reply:
I've moved `DateInput` and `Textarea` into new folders: `/components/dateInput` and `/components/textarea`